noxfile.py

```diff
@@ -10,6 +10,21 @@
 from typing import List
 
 import nox
+import toml
+
+
+def min_dependencies() -> List[str]:
+    """Extract the minimum dependencies from pyproject.toml.
+
+    Returns:
+        the minimun dependencies defined in pyproject.toml.
+
+    """
+    with open("pyproject.toml", "r", encoding="utf-8") as toml_file:
+        data = toml.load(toml_file)
+    dependencies = data.get("project", {}).get("dependencies", {})
+    return [dep.split(",")[0].replace(">=", "==") for dep in dependencies]
```

Comment:
wow,  that's a nice idea.  how about a check to make sure there's a `>=` or `==` in the first dep requirement?  then we can ensure pyproject has a standard format as well.

Reply:
Indeed, the suggested check is missing. Can you please explain me why you suggest to do it only for the first dep requirement  rather than all deps?

Reply:
I guess I mean the first version requirement for every package requirement i.e., for `dep.split(",")[0]`. :-)

Reply:
I suspect it but I wasn't sure :) Thanks! Will update it soon

Reply:
Updated. Also added a check for empty dependency list. Now it is bullet-proof :crossed_fingers: